Installation/test/Installation/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ else()
 endif()
 
 if(WITH_CGAL_Qt5)
-  find_package(Qt5 QUIET)
+  find_package(Qt5 QUIET COMPONENTS Core Widgets Xml OpenGL Gui)
```

Comment:
I wonder why we need so maybe components of Qt5. Should not `CGAL::CGAL_Qt5` take care of that by itself?

Reply:
Core, Widget, OpenGL and Gui are quite obvious, and Xml is for QDomElement that is required in some of the polyhedron demo. I think that specific test could do fine with only Core. I don't know if we can use CGAL_Qt5 without finding Qt5 though.